subiquity/common/types.py

```diff
@@ -801,3 +801,11 @@ class AdDomainNameValidation(enum.Enum):
 class AdPasswordValidation(enum.Enum):
     OK = 'OK'
     EMPTY = 'Empty'
+
+
+class AdJoinResult(enum.Enum):
+    OK = 'OK'
+    JOIN_ERROR = 'Failed to join'
+    EMPTY_HOSTNAME = 'Target hostname cannot be empty'
+    PAM_ERROR = 'Failed to update pam-auth'
+    UNKNOWN = "Didn't attempt to join yet"
```

Comment:
Are these strings displayed to the user? If so, they should be translatable.

Reply:
Based on the majority of the types definition in the same file, it doesn't seem they are displayed to the user.

Reply:
We are trending toward descriptive strings rather than what enum.auto() is generating, at minimum if you end up debugging at the API layer it's much nicer to look at the values rather than trying to puzzle out what integer maps to the return value.